src/containers/AddPalette/AddPalette.js

```diff
@@ -28,22 +30,30 @@ class AddPalette extends Component {
       color_4: activePalette[3],
       color_5: activePalette[4],
     }
-    this.props.handlePalette(url, addPaletteSuccess, "POST", newPalette)
+
```

Comment:
This is a creative way to handle checking that the user has followed all of the correct steps!